sql/000-ore.sql

```diff
@@ -87,169 +81,40 @@ CREATE FUNCTION compare_ore_64_8_v1_term(a ore_64_8_v1_term, b ore_64_8_v1_term)
-  NEGATOR = <>,
-  RESTRICT = eqsel,
-  JOIN = eqjoinsel,
-  HASHES,
-  MERGES
-);
-
-
-DROP OPERATOR IF EXISTS <> (ore_64_8_v1_term, ore_64_8_v1_term);
-
-CREATE OPERATOR <> (
-  PROCEDURE="ore_64_8_v1_term_neq",
-  LEFTARG=ore_64_8_v1_term,
-  RIGHTARG=ore_64_8_v1_term,
-  NEGATOR = =,
-  RESTRICT = eqsel,
-  JOIN = eqjoinsel,
-  HASHES,
-  MERGES
-);
-
-DROP OPERATOR IF EXISTS > (ore_64_8_v1_term, ore_64_8_v1_term);
-
-CREATE OPERATOR > (
-  PROCEDURE="ore_64_8_v1_term_gt",
-  LEFTARG=ore_64_8_v1_term,
-  RIGHTARG=ore_64_8_v1_term,
-  COMMUTATOR = <,
-  NEGATOR = <=,
-  RESTRICT = scalargtsel,
-  JOIN = scalargtjoinsel
-);
-
-DROP OPERATOR IF EXISTS < (ore_64_8_v1_term, ore_64_8_v1_term);
-
-CREATE OPERATOR < (
-  PROCEDURE="ore_64_8_v1_term_lt",
-  LEFTARG=ore_64_8_v1_term,
-  RIGHTARG=ore_64_8_v1_term,
-  COMMUTATOR = >,
-  NEGATOR = >=,
-  RESTRICT = scalarltsel,
-  JOIN = scalarltjoinsel
-);
-
-DROP OPERATOR IF EXISTS <= (ore_64_8_v1_term, ore_64_8_v1_term);
-
-CREATE OPERATOR <= (
-  PROCEDURE="ore_64_8_v1_term_lte",
-  LEFTARG=ore_64_8_v1_term,
-  RIGHTARG=ore_64_8_v1_term,
-  COMMUTATOR = >=,
-  NEGATOR = >,
-  RESTRICT = scalarlesel,
-  JOIN = scalarlejoinsel
-);
-
-DROP OPERATOR IF EXISTS >= (ore_64_8_v1_term, ore_64_8_v1_term);
-
-CREATE OPERATOR >= (
-  PROCEDURE="ore_64_8_v1_term_gte",
-  LEFTARG=ore_64_8_v1_term,
-  RIGHTARG=ore_64_8_v1_term,
-  COMMUTATOR = <=,
-  NEGATOR = <,
-  RESTRICT = scalarlesel,
-  JOIN = scalarlejoinsel
-);
-
-DROP OPERATOR FAMILY IF EXISTS ore_64_8_v1_term_btree_ops USING btree;
-
-CREATE OPERATOR FAMILY ore_64_8_v1_term_btree_ops USING btree;
-
-
-DROP OPERATOR CLASS IF EXISTS ore_64_8_v1_term_btree_ops USING btree;
-
-CREATE OPERATOR CLASS ore_64_8_v1_term_btree_ops DEFAULT FOR TYPE ore_64_8_v1_term USING btree FAMILY ore_64_8_v1_term_btree_ops  AS
-        OPERATOR 1 <,
-        OPERATOR 2 <=,
-        OPERATOR 3 =,
-        OPERATOR 4 >=,
-        OPERATOR 5 >,
-        FUNCTION 1 compare_ore_64_8_v1_term(a ore_64_8_v1_term, b ore_64_8_v1_term);
-
 -- Compare the "head" of each array and recurse if necessary
 -- This function assumes an empty string is "less than" everything else
 -- so if a is empty we return -1, if be is empty and a isn't, we return 1.
 -- If both are empty we return 0. This cases probably isn't necessary as equality
 -- doesn't always make sense but it's here for completeness.
 -- If both are non-empty, we compare the first element. If they are equal
 -- we need to consider the next block so we recurse, otherwise we return the comparison result.
-DROP FUNCTION IF EXISTS compare_ore_array(a ore_64_8_v1_term[], b ore_64_8_v1_term[]);
 
-CREATE FUNCTION compare_ore_array(a ore_64_8_v1_term[], b ore_64_8_v1_term[])
+
+CREATE FUNCTION compare_ore_array(a bytea[], b bytea[])
 RETURNS integer AS $$
   DECLARE
     cmp_result integer;
+    len_a integer;
+    len_b integer;
   BEGIN
+
+    SELECT array_length(a, 1) INTO len_a;
+    SELECT array_length(b, 1) INTO len_b;
```

Comment:
It doesn't look like `len_a` and `len_b` are used. Maybe remove them or change the code to use them.